lib/puppet/provider/gcompute_disk_type/google.rb

```diff
@@ -150,6 +135,7 @@ def self.resource_to_hash(resource)
   def resource_to_request
     request = {
       kind: 'compute#diskType',
+      deprecated: @resource[:deprecated],
```

Comment:
This looks wrong. Adding `output: true` to top-level deprecated resource should fix this.

Reply:
Done